features/basic-message/README.md

```diff
@@ -0,0 +1,77 @@
+# 0000: Basic Message
```

Comment:
The number should be updated to 0095.

Reply:
I suggest you add the word "Protocol" and the version "1.0" to the title at some point.

Reply:
The folder name needs to be changed from "basic-message" to "0095-basic-message". Then the index.md needs to be updated again.